draft-ietf-teep-architecture.md

```diff
@@ -735,11 +735,12 @@ Note that personalization data is not included in the table above.
 The use of personalization data is dependent on how TAs are used 
 and what their security requirements are. 
 
-TEEP requests from a TAM to a TEEP Agent can be encrypted with the
-TEE public key (to provide confidentiality), and are then signed with the TAM
+TEEP requests from a TAM to a TEEP Agent can be signed with the TAM
 private key (for authentication and integrity protection).
-Conversely, TEEP responses from a TEEP Agent to a TAM can be encrypted
-with the TAM public key, and are then signed with the TEE private key.  
+Conversely, TEEP responses from a TEEP Agent to a TAM can be 
+signed with the TEE private key. For encryption of the personalization
+data and the TA binary, the TA developer has to use public keys 
+unique to the TEE.
```

Comment:
"use" how?  This is confusing.  I prefer the old text, and line 745-746 only make sense with the old text, not this proposed change.

Reply:
Russ commented: 

> Section 5 says: "... encrypted with the TAM public key ...". This is not correct. The TAM public key is used to establish the key that is then used for encryption and decryption. Further the wording is aimed at RSA, which is a key transport algorithm. Please make the wording accommodate key transport and key agreement approaches.

However, I don't think this is addressed yet.

Reply:
Ming, can you do a wording suggestion here?

Reply:
Revised with suggestions. I kept the encryption clause for its purpose to provide end-to-end security without relying on network security such as use of TLS to exchange the TEEP messages.